modules/virt-customizing-storage-profile.adoc

```diff
@@ -109,23 +109,71 @@ status:
 <2> Specify the volume mode.
 <3> Specify the default cloning strategy.
 
-[id="virt-customizing-storage-profile-providers-and-default-behaviors_{context}"]
-.Storage providers and default behaviors
-
-[cols="1,1",options="header"]
-|===
-|Storage provider|Default behavior
-|rook-ceph.rbd.csi.ceph.com|Snapshot
-|openshift-storage.rbd.csi.ceph.com|Snapshot
-|csi-vxflexos.dellemc.com|CSI Clone
-|csi-isilon.dellemc.com|CSI Clone
-|csi-powermax.dellemc.com|CSI Clone
-|csi-powerstore.dellemc.com|CSI Clone
-|hspc.csi.hitachi.com|CSI Clone
-|csi.hpe.com|CSI Clone
-|spectrumscale.csi.ibm.com|CSI Clone
-|rook-ceph.rbd.csi.ceph.com|CSI Clone
-|openshift-storage.rbd.csi.ceph.com|CSI Clone
-|cephfs.csi.ceph.com|CSI Clone
-|openshift-storage.cephfs.csi.ceph.com|CSI Clone
-|===
\ No newline at end of file
+[id="virt-viewing-automatically-created-storage-profiles_{context}"]
+.Viewing automatically created storage profiles
```

Comment:
This should be a separate procedure with steps.

Reply:
@snarayan-redhat  There is only one step there - is it enough for a separate procedure?.

Reply:
You can call this section as `Viewing the storage profile details` and have 2 steps that are listed.
1. Getting list of profiles
2. Fetching single profile details

Reply:
@snarayan-redhat  I created a separate procedure module. Please take a look.